plugin/evm/vm_test.go

```diff
@@ -2268,6 +2301,110 @@ func TestTxAllowListSuccessfulTx(t *testing.T) {
 	}
 
 	require.Equal(t, signedTx0.Hash(), txs[0].Hash())
+
+	vm.clock.Set(reenableAllowlistTime.Add(time.Hour))
+
+	// Re-Submit a successful transaction
+	tx0 = types.NewTransaction(uint64(1), testEthAddrs[0], big.NewInt(1), 21000, big.NewInt(testMinGasPrice), nil)
+	signedTx0, err = types.SignTx(tx0, types.NewEIP155Signer(vm.chainConfig.ChainID), testKeys[0])
+	require.NoError(t, err)
+
+	errs = vm.txPool.AddRemotesSync([]*types.Transaction{signedTx0})
+	require.NoError(t, errs[0])
+
+	// accept block to trigger upgrade
+	blk = issueAndAccept(t, issuer, vm)
+	newHead = <-newTxPoolHeadChan
+	require.Equal(t, newHead.Head.Hash(), common.Hash(blk.ID()))
+	block = blk.(*chain.BlockWrapper).Block.(*Block).ethBlock
+
+	txs = block.Transactions()
```

Comment:
seems this assignment is not used. maybe you meant to add some assertion?

Reply:
removed